src/app/components/OmegaId/identification/OmegaId.tsx

```diff
@@ -0,0 +1,15 @@
+'use server'
+
+import OmegaIdElement from './OmegaIdElement'
```

Comment:
I think all of this needs a generalization to use qr-codes for skapres, registrations, ...

Reply:
Just remember that it is written I guess

Reply:
The next-qrcode library has a simple api. I don't think we need to make an abstraction on top of that. See `OmegaIdElement.tsx` for an example.